components/mosart/src/riverroute/RtmMod.F90

```diff
@@ -2233,19 +2234,19 @@ subroutine Rtmrun(rstwr,nlend,rdate)
           write(iulog,'(2a,i4,a,i10,i6)') trim(subname),' subcycling=',ns,': model date=',ymd,tod
        endif
      
-       if (inundflag) then
-          call t_startf('mosartr_inund_sim')
-          call MOSARTinund_simulate ( )
-          call t_stopf('mosartr_inund_sim')
-       else
+       !if (inundflag .and. wrmflag .eq. 0) then !use Luo's scheme when inundation is on and WM is off (keep it for now - tz)
+       !   call t_startf('mosartr_inund_sim')
+       !   call MOSARTinund_simulate ( )
+       !   call t_stopf('mosartr_inund_sim')
+       !else ! other cases
```

Comment:
delete commented out lines and re-indent code

Reply:
This is the old scheme. I intend to keep it for now as we may do some comparisons in the future.